lib/models/application.coffee

```diff
@@ -85,21 +87,22 @@ getApplicationModel = (deps, opts) ->
 	exports.getAll = (options = {}, callback) ->
 		callback = findCallback(arguments)
 
-		token.getUserId().then (userId) ->
-			return pine.get
-				resource: 'application'
-				options:
-					mergePineOptions
-						orderby: 'app_name asc'
-						filter:
-							user: userId
-					, options
+		auth.getUserDetails()
+			.then (body) ->
```

Comment:
Nit: I'd probably call this 'userDetails'

Reply:
> NIT: Most chained method calls in the SDK have the same indentation as their previous statement. Please reduce the indentation by 1 level.

and also as already said, `body` isn't that descriptive for this context.